src/snowflake/connector/arrow_context.py

```diff
@@ -119,22 +121,26 @@ def TIMESTAMP_LTZ_to_python_windows(
             )
             return time.localtime(microseconds)
 
-    def REAL_to_numpy_float64(self, py_double):
+    def REAL_to_numpy_float64(self, py_double: float) -> float64:
         return numpy.float64(py_double)
 
-    def FIXED_to_numpy_int64(self, py_long):
+    def FIXED_to_numpy_int64(self, py_long: int) -> int64:
         return numpy.int64(py_long)
 
-    def FIXED_to_numpy_float64(self, py_long, scale):
+    def FIXED_to_numpy_float64(self, py_long: int, scale: int) -> float64:
         return numpy.float64(decimal.Decimal(py_long).scaleb(-scale))
 
-    def DATE_to_numpy_datetime64(self, py_days):
+    def DATE_to_numpy_datetime64(self, py_days: int) -> datetime64:
         return numpy.datetime64(py_days, "D")
 
-    def TIMESTAMP_NTZ_ONE_FIELD_to_numpy_datetime64(self, value, scale):
+    def TIMESTAMP_NTZ_ONE_FIELD_to_numpy_datetime64(
+        self, value: int, scale: int
```

Comment:
dumb question: should it be float?

Reply:
https://github.com/snowflakedb/snowflake-connector-python/pull/1418#discussion_r1092324142